README.md

```diff
@@ -9,19 +9,23 @@
 
 ### Install Prerequisites
 #### Install Git
- * If you want to use Git in the command line, run `sudo apt install git`
+ * If you want to use Git in the command line, run `sudo apt install git`, you can get [Git for Windows](https://git-scm.com/) or use `choco install git`
  * If you're less terminal-inclined, you can install a Git GUI wrapper. [There's a nice list here.](https://git-scm.com/downloads/guis/)
     * I would personally recommend [GitKraken](https://www.gitkraken.com/). The free version has everyhting you need, (although their conflict resolver for Pro is nice,) and you can also get Pro free with [GitHub Education](https://education.github.com/)!
 #### <sup>AND</sup> Install C++
  1. Install a C++ compiler
     * the typical standard is `gcc`, you can install it with `sudo apt install gcc`
     * `clang` is an alternative that I feel has better error statements, you can install it with `sudo apt install clang`
- 2. Install CMake with `sudo apt install cmake`
+    * The typical standard compiler for Windows is [Microsoft Visual C++](https://www.visualstudio.com/)
+    * [MinGW](https://www.msys2.org/) or [Cygwin](https://www.cygwin.com/) are valid alternatives if you like GCC, oh there's also [WSL](https://docs.microsoft.com/en-us/windows/wsl/install-win10) as well ^^.
+	
+ 2. Install CMake with `sudo apt install cmake` or by downloading it from [CMake](https://cmake.org/download/) or get it with `choco install cmake`.
 #### <sup>AND</sup> Install Libraries
- * Debian-based (Ubuntu) `sudo apt install build-essential libboost-all-dev libssl-dev libcrypto++-dev libcurl4-openssl-dev`
- * Fedora/RHEL-based `sudo dnf install openssl-devel boost-devel cryptopp-devel cryptopp`
+ * Debian-based (Ubuntu) `sudo apt install build-essential libboost-all-dev libssl-dev`
+ * Fedora/RHEL-based `sudo dnf install openssl-devel boost-devel`
+ * Windows Get [vcpkg](https://github.com/microsoft/vcpkg), follow the page instruction to install it, then type`vcpkg install boost-system boost-beast openssl`
```

Comment:
Grammar here feels awkward. Maybe we need a `tools` section for things like `git`, `vcpkg`, `cmake`, and `choco`?

Reply:
Oh, and maybe something about how `vcpkg` is cross-platform?

Reply:
Should I just point vcpkg in all platforms or say something like you can use vcpkg on linux/mac as well?

Reply:
That would be good